modules/service/service.tf

```diff
@@ -21,6 +21,15 @@ resource "aws_ecs_service" "service" {
     assign_public_ip = false
   }
 
+  dynamic "deployment_circuit_breaker" {
+    for_each = var.deployment_circuit_breaker ? ["single"] : []
+
+    content {
+      enable   = true
+      rollback = var.deployment_circuit_breaker_rollback
+    }
+  }
+
```

Comment:
Do we need a dynamic block? Could we pass the value of `var.deployment_circuit_breaker` into `enable`?

Reply:
I thought it might prevent us getting into a strange situation with `deployment_circuit_breaker_rollback` being true and `deployment_circuit_breaker` being false, but apparently the ECS API doesn't care if you set rollback to true and do not have the circuit breaker enabled, so lets chuck the dynamic block 👍